src/cartridges/int_adyen_SFRA/cartridge/client/default/js/applePayExpress.js

```diff
@@ -125,8 +121,9 @@ function callPaymentFromComponent(data, resolveApplePay, rejectApplePay) {
 
 initializeCheckout()
   .then(async () => {
+    const paymentMethodsResponse = await getPaymentMethods();
```

Comment:
maybe its good to put this as part of the initializeCheckout function also in a try catch so we are sure there always will be values